From 5c4bc807f712031cfb8243afa96016a194947572 Mon Sep 17 00:00:00 2001
From: Sam Lantinga <slouken@libsdl.org>
Date: Wed, 30 Nov 2022 05:59:56 -0800
Subject: [PATCH 050/458] Revert "SDL_test_md5.h: fix MD5UINT4 type to be
 really 32 bits."

This reverts commit 8eeca8c7db323d82ec257db9886e2ba16c0c892a as it's potentially an ABI break.
---
 include/SDL_test_md5.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/SDL_test_md5.h b/include/SDL_test_md5.h
index 5d4c2e679..17b1d2be7 100644
--- a/include/SDL_test_md5.h
+++ b/include/SDL_test_md5.h
@@ -65,7 +65,7 @@ extern "C" {
 /* ------------ Definitions --------- */
 
 /* typedef a 32-bit type */
-  typedef Uint32 MD5UINT4;
+  typedef unsigned long int MD5UINT4;
 
 /* Data structure for MD5 (Message-Digest) computation */
   typedef struct {
-- 
2.17.1

